### ISSUE TITLE
AGs - Fehlende Leitlinien und fehlende Sichtbarkeit

### ISSUE THREAD

**Issue by Mike-wall-pirat** [sebulino/Strategie#7]
Abgesehen von der Tatsache, das viele AGs tot sind, fehlen vorallem Leitlinien. Auch hier ein Beispiel:
Ich habe versucht, selbst eine AG zu gründen bzw. diese zu leiten. Ich bin krachend gescheitert. Grund ist, das ich mir zum einen ein Thema ausgesucht habe, das ich selbst nur managen könnte, aber nicht selber bearbeiten. Und zum anderen, weil die von mir auserkorenen 2 Mitarbeiter selbst zwar interessiert waren, aber ohne Anleitung nichts gemacht haben. Ergebnis, es ist nicht passiert und ich hab die Idee der AG zu Grabe getragen. Andere AGs, die ich z.B. zwecks Informationsaustausch angeschrieben habe, haben obwohl aktiv, nicht auf meine Emails reagiert.
Was meine ich mit Leitlinien?
In den Leitlinien wird das grundsätzliche einer AG definiert. Selbstständig arbeiten, Arbeit dokumentieren, Ergebnisse dokumentieren, Erkenntnisse/Wissen dokumentieren und kommentieren. Auf der Basis von letzterem z.B. Infomaterial für Mandatsträger erstellen. Empfehlungen ausprechen etc. Wenn das passiert, ist eine AG auch sichtbar! Ohne das ist sie Sinn- und nutzlos. Ich habe solche Informationsmaterialien im Bereich der Elektromobilität für unsere Mandatsträger erstellt und verteilt. Fehlen tut auch eine Stelle, die diese Ergebnisse dann eben an z.B. alle Mandatsträger verteilt!

**Comment by sebulino** [comment]
Ja, das ist genau die Aufgabe, die in dem Abschnitt "Problemfeld politische Arbeit" angesprochen ist. Ein Teil der Regeln wurde mal hier zusammengefasst https://wiki.piratenpartei.de/Koordinatorenkonferenz/AG-Regeln aber klar, es ist da notwendig das zu überarbeiten. Und führt über zu dem Thema Innerparteiliche Bildung, wo es vermutlich sinnvoll wäre zu erklären, wie Arbeit in den AGs aussieht. 

Ich hatte in der Vergangenheit bereits auch schon mal was angestoßen, was auf den Aspekt der Dokumentation eingeht, den du nennst. Das sah ungefähr so aus https://pad.sebulino.com/s/B1XUKVkSQ , Handlungsnotwendigkeit sehe ich noch immer. 

Ich würde mir dann einfach erlauben auf dich zuzukommen, wenn es darum geht in einer Gruppe da erneut Wege aufzuzeigen :D

**Comment by sebulino** [comment]
Hier eine Ergänzung aus einem anderen Issue
https://github.com/sebulino/Strategie/issues/16#issue-488698898
